games.php

```diff
@@ -4,18 +4,25 @@
 
 include('./config.php');
 
-function order_servers($a, $b)
+function order_servers($servers)
 {
-    // Find the offset of the first alphabetic character in the server name
-    preg_match('~[a-z]~i', $a['name'], $a_start, PREG_OFFSET_CAPTURE);
-    preg_match('~[a-z]~i', $b['name'], $b_start, PREG_OFFSET_CAPTURE);
-
-    // Compare the alphabetic portion only, ignoring special characters at the start
-    // Servers without any alphabetic characters are pushed to the end
-    $a_comp = $a_start ? substr($a['name'], $a_start[0][1]) : "z" . $a['name'];
-    $b_comp = $b_start ? substr($b['name'], $b_start[0][1]) : "z" . $b['name'];
-
-    return strcasecmp($a_comp, $b_comp);
+	// Create an array of server names, with any non-alphanumeric characters replaced with z
+	$names = array();
+	foreach($servers as $key => $server){
```

Comment:
Style nit: can we please have a space between the `foreach (`, and the OpenRA coding style is to omit braces on single line conditionals.

Reply:
Ok, this has now been updated.